src/index.tsx

```diff
@@ -249,6 +281,9 @@ export function generateTrigger(
 
     onMouseEnter = e => {
       const { mouseEnterDelay } = this.props;
+      if ((e.target as any).disabled) {
+        this.hideOnMouseOut = e.target as Element;
```

Comment:
Could we use state to save this instead? Since class variable may miss catch by concurrent mode in future React released.

Reply:
@zombieJ Sure.